com.editor.gamma.json

```diff
@@ -0,0 +1,56 @@
+{
+    "app-id": "com.editor.gamma",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "command": "/app/bin/gamma",
+    "finish-args": [
+    	"--share=network",
+    	"--socket=x11",
+		"--share=ipc",
+		"--socket=session-bus",
+		"--filesystem=host",
+		"--own-name=com.editor.gamma"
+    ],
+    "modules": [
+    	{
+            "name": "gtksourceview",
+            "config-opts": [ "--disable-Werror" ],
+            "sources": [
+            {
+                "type": "archive",
+                "url": "https://download.gnome.org/sources/gtksourceview/4.0/gtksourceview-4.0.0.tar.xz",
+                "sha256": "6e5c7a28a7fa456a89f289f0659c57100cc09fe692db6abfcc23f7ade5d5b32a"
+            }
+            ]
+        },
+        {
+            "name": "metainfo",
+            "buildsystem": "simple",
+            "build-commands": [
+            	"install -Dm644 ${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo/"
```

Comment:
The metainfo should be part of the upstream repository. It's not a flatpak specific file.

Reply:
Ok, I will move it back to upstream repo.